### ISSUE TITLE
problem of training both models of classifier_model annd Randomwalk_Word2vec 

### ISSUE THREAD

**Issue by intellectwood** [ma-compbio/Hyper-SAGNN#12]
Hello, I am very admiring your work.But  I notice last line of main.py can train both model of classifier_model and Randomwalk_Word2vec but with loss=1, loss2=0 the code only train model of classifier_model, right?

Then here is my question,  setting loss=1, loss2=1, code got error:
![image](https://github.com/ma-compbio/Hyper-SAGNN/assets/30440402/f7876c6b-59db-4c24-a6f6-6622d6c0d5bf)

And I want to know whether only one model of classifier_model perform better or both model?

**Comment by ruochiz** [comment]
Hi, Thank you for your interest. Yeah, with the newer version of tensorflow, I stopped maintaining the code for the random walk part. The answer is that if you use the model with the adj mode, then just the classifier_model would work well. If it's random walk based, then that part of the loss would also be preferred. A quick fix, might just be changing line 132 of main from 
```
example_emb = model.forward_u(examples)
```

to 
```
example_embed, _ = model.forward_u(examples)
```

In addition, main_torch.py get rid of the dependencies of tensorflow 1.0 which would be slightly more up to date (while losing support for the random walk part of the model)

**Comment by intellectwood** [comment]
Still can't run, got this error, have you tried?
![image](https://github.com/ma-compbio/Hyper-SAGNN/assets/30440402/05e93b5c-dd6a-41e1-9447-fb41b13ae3fe)
Then I am interested in your new version removing random walk, is it because with the adj mode can show best result or training both models(classifier_model annd Randomwalk_Word2vec) improves little?

**Comment by zhangwen123123** [comment]
你好，你复现出来这两个值有这么小吗auc： 0.499， aupr： 0.166 还是我哪个环节出错了
